tests/backends/aiida_sqlalchemy/migrations/django_branch/test_migrate_to_head.py

```diff
@@ -7,21 +7,12 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
-# pylint: disable=invalid-name
-"""Database migration."""
```

Comment:
Wait, how does GitHub think an actual Django migration file was renamed to a test file? Probably misinterpreted. It simply got deleted the migration, didn't it?

Reply:
yeh its an entirely new file, github is drunk